src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -288,11 +310,13 @@ private Request make(final byte[] boundary) throws IOException {
          * Copy until boundary reached.
          * @param target Output file channel
          * @param boundary Boundary
+         * @param body Origin request body
          * @throws IOException If fails
          * @checkstyle ExecutableStatementCountCheck (2 lines)
          */
         private void copy(final WritableByteChannel target,
-            final byte[] boundary) throws IOException {
+            final byte[] boundary,  final  ReadableByteChannel body)
+                throws IOException {
```

Comment:
@carlosmiranda thank you, could you help me more here:
 if you mean line 319: [ERROR] /src/main/java/org/takes/rq/RqMultipart.java[317]: Line is longer than 80 characters (found 87). (LineLengthCheck)
317 + splitted 318 - longer than 82